ui/components/app/modals/account-details-modal/account-details-modal.component.js

```diff
@@ -6,8 +6,11 @@ import AccountModalContainer from '../account-modal-container';
 import QrView from '../../../ui/qr-code';
 import EditableLabel from '../../../ui/editable-label';
 import Button from '../../../ui/button';
-import { getURLHostName } from '../../../../helpers/utils/util';
-import { isHardwareKeyring } from '../../../../helpers/utils/hardware';
+// eslint-disable-next-line import/no-duplicates
```

Comment:
Can this import be merged?

Reply:
yes, changed in d9c01765203c15f163465c62559d658151ce3942